docs/user_guide/metrics.md

```diff
@@ -133,36 +133,40 @@ If building Triton locally, the `TRITON_ENABLE_METRICS_CPU` CMake build flag can
 
 ## Response Cache Metrics
 
-Compute latency metrics in the [Inference Request Metrics table](#inference-request-metrics) above are calculated for the
-time spent in model inference backends. If the response cache is enabled for a
-given model (see [Response Cache](response_cache.md)
+Cache metrics can be reported in two ways:
+
+1. A base set of cache metrics will be reported
+by Triton directly, such as the cache hit/miss counts and durations described
+below.
+
+2. As of 23.02, additional cache metrics may be reported depending on the 
+[cache implementation](response_cache.md#cache-implementations) 
+being used through Triton's [Metrics API](#custom-metrics).
```

Comment:
So, if they use custom metrics with the same metric name and type that we use in the local cache, all the tooling that uses these values (e.g., perf analyzer) will report correct numbers?

Reply:
The minimal set that remains in Triton-core and in this doc (per-model cache_hit_count/duration, cache_miss_count/duration) is used by perf analyzer.

As with any existing metrics, it's possible for a user to update their values with the metrics API - though they should not do this. I mention a couple concerns / potential solutions around this in [DLIS-4072](https://jirasw.nvidia.com/browse/DLIS-4072), but for now it's essentially up to the user to not pick duplicate names.